Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+ADD . /project
+WORKDIR /project
+
+RUN apt-get -y update
+RUN apt-get -y install postgresql-client
```

Comment:
I suspect thumbnailing wont work, because the jpeg/gif/png whatever libs are not installed.

We install these packages on the aldryn base image: https://github.com/aldryncore/base-docker/blob/master/stack/packages.txt

I don't know by heart which ones are needed. definitly ``libjpeg-dev`` and ``libjpeg62``. But also some others.